### ISSUE TITLE
Single line comments break syntax highlighting

### ISSUE THREAD

**Issue by AlexKeySmith** [microsoft/vscode-mssql#1253]
- MSSQL Extension Version: 1.5.0

Visual Studio Code details

Version: 1.34.0 (system setup)
Commit: a622c65b2c713c890fcf4fbf07cf34049d5fe758
Date: 2019-05-15T21:59:37.030Z
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17134

Steps to Reproduce:

When using single line comments i.e. `--` they are not syntax highlighted and often seem to report errors on subsequent lines. `/**/` multi-line comments seem to work fine though. 

I've tested this connected to LocalDB and Azure SQL DWH.

![image](https://user-images.githubusercontent.com/912653/60102974-a142c500-9756-11e9-8f42-3ffd03934707.png)

**Comment by inspektor99** [comment]
I get the same thing.  I have to disable, restart VS Code, enable, and it fixes the issue.  It's annoying, I have to do this every time I launch VS Code.

fresh vs code start
![image](https://user-images.githubusercontent.com/1494954/63110227-c32f1b80-bf3f-11e9-8b42-a1e476b4eb6f.png)

disable/enable plug in
![image](https://user-images.githubusercontent.com/1494954/63110366-1b661d80-bf40-11e9-8596-947f7412260c.png)

**Comment by TarekSalha** [comment]
I do have the same problem. also other comments from different languages are highligthed instead.
these are "//" and "#"
that can become annoying, if you are writing both c#, python and sql. Then the possibility to confuse them while being in the tunnel is high.

**Comment by TarekSalha** [comment]
I can see some improvements with the syntax highligting up until now. However, this specific problem still persists and it would really add a lot of value if it would be fixed. Reason is, that most developers write lots of comments to explain their code. It's hard to distingiush that from real code. Currently, I am copying the code to Data Studio to read it properly.

**Comment by DavidP-Vitro** [comment]
@TarekSalha While it won't help with "#" being highlighted as a comment when it shouldn't be, in order to get comments to display somewhat correctly, I've been using "--//" for one line comments as a workaround to get them colored correctly.

**Comment by TarekSalha** [comment]
> @TarekSalha While it won't help with "#" being highlighted as a comment when it shouldn't be, in order to get comments to display somewhat correctly, I've been using "--//" for one line comments as a workaround to get them colored correctly.

funny workaround, I will remember it

**Comment by DavidP-Vitro** [comment]
This might not be an issue with this extension, but rather another extension conflicting and getting precedence. I opened the developer tools console and noticed that the SQL language grammar was being overwritten by the [Azure Data Lake Tools](https://github.com/microsoft/AzureDatalakeToolsForVSCode) extension which provides support for U-SQL programming. 
![image](https://user-images.githubusercontent.com/41449057/80635516-f6f89580-8a29-11ea-96e6-d0dd918ef4c2.png)

As soon as I disabled that extension, VSCode reloaded the grammar from this extension and the syntax highlighting was correct.
